docs/tutorial.md

```diff
@@ -0,0 +1,183 @@
+# Oria Operator Tutorial
+
+### An in-depth walkthrough of building and running a oria-operator with memcached-operator.
+
+## Prerequisites
+
+- [Operator SDK](https://sdk.operatorframework.io/docs/installation/) v1.8.0 or newer
+- User authorized with `cluster-admin` permissions.
+- [GNU Make](https://www.gnu.org/software/make/)
+
+## Overview
+
+We will create a sample project to let you know how it works and this sample will:
+
+- Create a Memcached-Operator Deployment using this [tutorial](https://sdk.operatorframework.io/docs/building-operators/golang/tutorial/)
+- Generate the manifests using `config/default`
+- Run the Operator and after success, delete the `ClusterRole` and `ClusterRoleBinding`. Now, the `memcached-operator` will start complaining about it.
+- Create similar ClusterRole template in ScopeTemplate and assciate it with ScopeInstance
+- Run the `oria-operator`, it will create `ClusterRole` and `ClusterRoleBinding`. This will make sure that `memcached-operator` will not complain about RBAC's. 
+
+### Create a new project
+
+Use the [Operator SDK](https://sdk.operatorframework.io/docs/installation/) CLI to create a new memcached-operator from the [tutorial](https://sdk.operatorframework.io/docs/building-operators/golang/tutorial/).
```

Comment:
We should mention that they don't have to read the entire tutorial from SDK. Only to reference it for more details, but below we will be showing them what they need for the purposes of this demo. 

Not quite sure how to rephrase it, but as it reads now it seems like they MUST follow the SDK tutorial first BEFORE doing anything on the oria side. I don't think that's what you intended.

Reply:
Yes, I added all the steps and pointed SDK tutorial just for information.

Reply:
updated.